packages/booter-lb3app/src/lb3app.booter.ts

```diff
@@ -75,6 +75,8 @@ export class Lb3AppBooter implements Booter {
     debug('Loading LB3 app from', this.appPath);
     const lb3App = require(this.appPath);
 
+    debug('Make sure loopback-boot@3.2.1 or higher is installed');
```

Comment:
As a user seeing this message, I think I would be confused. If I am already using supported loopback-boot version, then why am I being told to use it?

Are you trying to offer advice for users that may observe that application never finishes to boot because they are using an old versions of loopback-boot? If that's the case, then I am proposing a slightly different message.

```suggestion
    debug('If your LB3 app does not boot correctly then make sure it is using loopback-boot version 3.2.1 or higher.');
```

Thoughts?

Reply:
Yes, I agree this is better. I'll make the change. 👍